pkg/baseline/packageIndex.json

```diff
@@ -9,8 +9,9 @@
       "BaselineVersion": "4.4.0",
```

Comment:
@ericstj I know you have signed off and thanks... but should this be 4.5.0 like others?